hatg/addons/functions/functions/conditions/fn_surfaceIsGrass.sqf

```diff
@@ -29,12 +29,16 @@ private _surfaceTexture = toLowerANSI (surfaceTexture _pos);
 [_surface, 2, _fnc_scriptName] call HATG_fnc_log;
 [_surfaceTexture, 2, _fnc_scriptName] call HATG_fnc_log;
 
-private _surfacesNames = ["grass", "forest", "thorn", "field", "hlina", "trava"];
+private _surfacesNames = ["grass", "forest", "thorn", "field", "hlina", "trava","gbr_es_plevel1","gbr_es_tierra1","gbr_es_tierra2","gbr_es_tierra3","gbr_es_tierra10"];
```

Comment:
Rather than giving absolute classnames of surface types, could you not just add `"plevel", "tierra"`?

In the case that only some of the `type123` numbers are actually grass, it would still be better to do the above and make the blacklist actually useful